dataplane/saiserver/ports.go

```diff
@@ -92,6 +92,7 @@ func getPreIngressPipeline() []*fwdpb.ActionDesc {
 func getL3Pipeline() []*fwdpb.ActionDesc {
 	return []*fwdpb.ActionDesc{
 		fwdconfig.Action(fwdconfig.LookupAction(IngressActionTable)).Build(),                                                                                    // Run ingress action.
+		fwdconfig.Action(fwdconfig.LookupAction(invalidIPTable)).Build(),                                                                                        // Do not forward packets with invalid addresses
```

Comment:
Should this check happen before IngressActionTable?

Reply:
i think the desired behavior is that an ingress ACL rule could punt the packet before it's dropped, so this needs to happen after